Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2.7-slim
+
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir /genomics-workspace
+
+WORKDIR /genomics-workspace
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y libsqlite3-dev
```

Comment:
@deming7h777 I agree with Codacy on this issue.  Why are you running upgrade and update here instead of just making sure the required versions are installed?